concurrent/actor/eventloop/LogicalTimerQueue.java

```diff
@@ -1,86 +0,0 @@
-/*
```

Comment:
This class is an implementation detail of `EventLoop`, and therefore has been moved there as an inner class.

Given that the only usage of `LogicalTimerQueue<V>` is for storing scheduled jobs, it has been a) renamed to `ScheduledJobQueue` and b) made to no longer have the generic parameter.